controllers/comments_controller.js

```diff
@@ -0,0 +1,22 @@
+const { createComment } = require("../models/comments_model");
+const { articleId } = require("../models/articles_model");
+
+const postComment = (request, response, next) => {
+  const { article_id } = request.params;
+  const { username, body } = request.body;
+  
+
+  const promises = [
+    articleId(article_id),
```

Comment:
Good thinking with this check but because it is a foreign key PSQL would throw an error if we tried to insert a comment with an article id that didn't exist anyway . I wouldn't worry about going back to change this, but definitely bare this in mind when you come to refactor.